Makefile

```diff
@@ -14,6 +10,19 @@ check-go:
 	@./scripts/check-go.sh
 	@echo
 
+check-reqs:
+	@./scripts/check-reqs.sh
+	@echo
+
```

Comment:
can you remove one extra line here?

Reply:
sure

Reply:
@Madhu-1 but I dont see it

Reply:
@humblecI see 2 empty lines [line 16 and line 17]

Reply:
ah. yes, done. Thanks